### PR TITLE
📝 Document all the features

### LINKED ISSUE #83: 📝 Document DB design
## Description

Document the DB Design
- Cover decisions and special handling
- Multi-tenancy
- Adding things to user model

## Affects

- Documentation

## Consequences

Users of template won't know what's available

### PR DESCRIPTION
**Pull request by craineum**
# Pull Request

## Story

Fix #83 #60 #82 #81 #69 

## Description

Document features

## Affected Areas

Readme documentation

## Definition of Done

- [x] README updated
- [ ] Code reviewed
- [ ] Feature accepted

## PR Comments Emoji Legend

😻 - `:heart_cat_eyes:` Compliment to code/idea/etc
♻️ - `:recycle:` Non-blocking proposed refactor
➕ - `:heavy_plus_sign:` Non-blocking proposed addition
🔥 - `:fire:` Non-blocking proposed removal
❓ - `:question:` Non-blocking question
⚠️ - `:warning:` Blocking comment that must be addressed before PR

## Gifs for life (required)

mandatory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +23,131 @@ The following is a list of frameworks, utilities, libraries, components, etc...
...
+When an administrator creates an account, the new user will receive an email that contains a link to the application for them to establish an account. On the backend, when the account is created, it is created with a random password and a password reset token is set.
+
+Clicking the link in the email will take them to a page to create a password for their account and then allows them to use the application. This clears the token and sets a new password in the backend.
+
+### Multi-Tenancy
+
+Multi-Tenancy is the ability for the application to have users be separated by different tenants. This way one tenant could be kept hidden from another. The reason we support this out of the box is it's difficult to add in later if the data isn't structured in a way to support it.
+
+While we haven't implemented much with multi-tenancy, we do take care of it when creating and maintaining accounts. The data is structure in a way that will make it easier to implement multi-tenant solutions.
+
```
Comment (house9) [inline README.md:66]: `The data is structure` -> The data is structured

**Review by house9 — APPROVED** [review]
LGTM

came across one possible typo

File: README.md [hunk 2]
```diff
@@ -22,6 +23,131 @@ The following is a list of frameworks, utilities, libraries, components, etc...
...
+This is a brief description of your testing frameworks and the philosophy behind how and what we test. Over all we put most of our testing effort into unit tests while we focus on happy path for our feature tests.
+
+We use the built in setup that [RedwoodJS](https://redwoodjs.com/docs/testing#redwood-and-testing) has implemented on the unit testing side and have implemented [Playwright](https://playwright.dev/) on the feature testing side
+
+#### Unit Testing
+
+We strive to testing things in isolation and not test anything that is tested via other means. This is especially true for any dependencies we may be using. While we aren't going for 100% coverage we do want to cover as much as we can. We also want to provide placeholders and patterns for future development to easily continue.
+
+Tests are located next to the files they are testing. Making it easy to find associated tests. We nest `describe` blocks for context when certain setup or data is required. Some tests are placeholders, in that they only make sure the thing being tested doesn't throw any errors. These are in place to add tests when the code becomes more involved.
```
Comment (zan-rosenthal) [inline README.md:122]: Maybe add an example of the naming pattern for a file and for a test here.

File: README.md [hunk 3]
```diff
@@ -22,6 +23,131 @@ The following is a list of frameworks, utilities, libraries, components, etc...
...
+- Run with: `yarn test`
+  - Spins up a postgres docker container for the test database
+  - Runs all tests in watch mode
+  - See root `package.json` for more commands
+- Global mocks are made available via `*.mock.ts` files
+  - Make sure you don't include anything that shouldn't be globally mocked
+
+#### Feature/End-to-End Testing
+
+Feature testing is brittle and a pain to maintain. In order to make this easier we only keep happy path tests around. We also write tests for development that may not be happy path, that we then delete after development is done.
```
Comment (zan-rosenthal) [inline README.md:139]: Do the development tests get committed? If users of the template don't see them, do we need this sentence?

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]

**Review by zan-rosenthal — APPROVED** [review]

**Pull request merged**